### PR TITLE
Fix error when delete by a decimal primary key column.

### PR DESCRIPTION
**Pull request by Crossoverrr**
Fix error when delete by a decimal primary key column.

### CODE REVIEW

**Comment by Crossoverrr** [comment]
Screenshot from 2020-08-04 18-21-06
Screenshot from 2020-08-04 18-22-30

File: presto-kudu/src/test/java/io/prestosql/plugin/kudu/TestKuduIntegrationDecimalColumns.java [hunk 1]
```diff
@@ -80,6 +80,23 @@ public void testDecimalColumn()
...
+        assertUpdate("CREATE TABLE IF NOT EXISTS test_decimal (" +
+            "decimal_id decimal(18, 3) WITH (primary_key=true), " +
+            "col_decimal decimal(18, 3)" +
+            ") WITH (" +
+            " partition_by_hash_columns = ARRAY['id'], " +
+            " partition_by_hash_buckets = 2" +
+            ")");
+
+        assertUpdate("INSERT INTO test_decimal VALUES (1.1, 1.1), (2.2, 2.2), (3.3, 3.3)", 3);
+        assertUpdate("DELETE FROM test_decimal WHERE decimal_id = 2.2", 1);
```
Comment (wendigo) [inline presto-kudu/src/test/java/io/prestosql/plugin/kudu/TestKuduIntegrationDecimalColumns.java:95]: You could assert here on SELECT to test whether a row was actually deleted.

Comment (Crossoverrr) [reply presto-kudu/src/test/java/io/prestosql/plugin/kudu/TestKuduIntegrationDecimalColumns.java:95]: Yes, that's cool, thank you!
Actually I think when it return 1 in `assertUpdate` it means the delete action is succeed.    :)

Comment (Crossoverrr) [reply presto-kudu/src/test/java/io/prestosql/plugin/kudu/TestKuduIntegrationDecimalColumns.java:95]: @wendigo Thank you for your suggestion, I have added the `select` sql.   :)

**Comment by Crossoverrr** [comment]
> Thanks for working on this. Kindly fix the checkstyle issue (for `TestKuduIntegrationDecimalColumns`)

Thank you for your help! I have fix  the checkstyle problem.    :)

**Pull request merged**